src/Symfony/Component/Cache/Adapter/TagAwareAdapter.php

```diff
@@ -39,125 +39,68 @@ class TagAwareAdapter implements TagAwareAdapterInterface, TagAwareCacheInterfac
     use LoggerAwareTrait;
 
     public const TAGS_PREFIX = "\0tags\0";
-    private const ITEM_PREFIX = '$';
-    private const TAG_PREFIX = '#';
-    private const MAX_NUMBER_OF_KNOWN_TAG_VERSIONS = 1000;
```

Comment:
I might have been a bit abrupt reverting so many changes. That's my way to adopt your changes :)
Let's discuss them one by one.

Here, you're adding a limit to the number of know tag versions and that's a new behavior, right?

Reply:
OK, reintroduced now.